teleport-bootstrap-script/metadata.tpl

```diff
@@ -2,10 +2,20 @@
 
 set -e
 
+get_private_ip () {
+  PRIVATE_IP="$(curl -s http://169.254.169.254/latest/meta-data/local-ipv4)"
+  if [ $? != 0 ]; then
+    PRIVATE_IP="$(hostname -I | tr ' ' '\n' | grep -m 1 -E '^(192\.168|10\.|172\.1[6789]\.|172\.2[0-9]\.|172\.3[01]\.)')"
```

Comment:
perhaps add a comment here to explain this a bit

Reply:
done